### ISSUE TITLE
Unable to join 1.20.2 server on 1.20.1 client

### ISSUE THREAD

**Issue by Libreh** [ViaVersion/ViaFabric#275]
### `/viaversion dump` Output

https://dump.viaversion.com/e97266abb81866ad38830e9119bf77829566f77852a01d4bf3ae3a85771900b5

### Console Error

```
https://paste.gg/p/anonymous/00e93cd78dac4bffb6b75a578103429b
```

### Bug Description

Unable to join 1.20.2 server on 1.20.1 client

### Steps to Reproduce

1. Join 1.20.2 server using 1.20.1 client
2. Not connect because of error

### Expected Behavior

The client should be able to successfully connect to the server

### Additional Server Info

I'm not using a proxy
![error](https://github.com/ViaVersion/ViaFabric/assets/103264719/a6cce7f4-2515-4b22-bf19-150965e9ea17)

### Checklist

- [ ] I have checked if this is specific to ViaFabric platform, and doesn't happen when running it on other platforms (Paper/Sponge/Velocity/Waterfall).
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried the latest build(s) from https://ci.viaversion.com/ and the issue still persists.

**Comment by luavixen** [comment]
Had this same issue with a large server, had to downgrade to 1.20.1. Can reproduce.

**Comment by Kichura** [comment]
Confirmed bug.

**Comment by FlorianMichael** [comment]
Can you try the version on the dev branch?

**Comment by Kichura** [comment]
I have tested the dev branch and it manages to persist here.

**Comment by uricot** [comment]
Same issue here

**Comment by Kichura** [comment]
Update: This has turned out to be a [fabric API problem](https://github.com/orgs/FabricMC/discussions/3391), although i'm unsure when this will be resolved at all; 1.19.4 servers and older however aren't affected.

**Comment by GrueFinder** [comment]
> Update: This has turned out to be a [fabric API problem](https://github.com/orgs/FabricMC/discussions/3391), although i'm unsure when this will be resolved at all; 1.19.4 servers and older however aren't affected.

Are there any more updates on this? You're saying it's a Fabric API problem, but the Fabric API team is saying it's an (unreported) ViaBackwards problem, and at the moment it doesn't seem like anyone is taking ownership of this.

**Comment by Kichura** [comment]
> > Update: This has turned out to be a [fabric API problem](https://github.com/orgs/FabricMC/discussions/3391), although i'm unsure when this will be resolved at all; 1.19.4 servers and older however aren't affected.
> 
> Are there any more updates on this? You're saying it's a Fabric API problem, but the Fabric API team is saying it's an (unreported) ViaBackwards problem, and at the moment it doesn't seem like anyone is taking ownership of it.

One of the methods we have thought of is attempting to inject into Fabric's API or/and block fabric's payloads in viaversion but this is ultimately the worst solution out of anything else, but we are open to ideas to how to resolve this for both sides.

**Comment by SyedOmarNoor** [comment]
I was thinking of trying to replace the networking jar in fabric api with a pre-3224 one and seeing if that works, might try later (iirc the api is referenced by the mods so changing it at the api file should affect the mods as well)

**Comment by SyedOmarNoor** [comment]
so i tried just doing that and it doesnt seem to help? i put the 88.5 networking into 90.7

**Comment by RealKomander** [comment]
Looks like I have the same issue, have you contacted fabricAPI team? Have they done anything about it?

**Comment by Kichura** [comment]
Fixed in [d30766a](https://github.com/ViaVersion/ViaBackwards/commit/d30766a17a1d9ad55778ceeb3bf511c7385b5ed0).
